### DISCUSSION TITLE
is this project abandoned?

### DISCUSSION THREAD

**Discussion by netllama** [bstansell/conserver#78] [General]
I don't see any updates, or changes since last September (2021).  New issues submitted with no replies.

**Comment by bstansell** [comment]
I wouldn't say "abandoned", but it certainly isn't getting the attention I used to give it.

There are a couple "easy" things here that should be taken care of (by me, and I've failed to). In some cases, I'm hoping community members can chime in and help.  And some things, I'm not sure I can help with. Silence isn't the answer, of course.

On the flip side, for a ton of use cases, the code is "stable". Or, at least, has been. Arguments can be made if it is no longer the case...some of the libraries that can be linked in have evolved and that can certainly be causing trouble. 

As always, I'm open to pull requests, for anyone daring enough to jump into that layer. I need to actually respond to them in a timely fashion, of course.

**Comment by netllama** [comment]
There's no shame in not having the time (or even interest) to maintain this code base.  However, I do feel that proper expectations should be set somewhere so that those who have come to depend on this important project know what to expect if they run into bugs, and anyone considering using it also is aware that they may be on their own, or have a long wait for a response to their issue.

What I do know is that my organization is using this software for many thousands of servers, and we are seeing a number of odd problems, that I'm tempted to assume are corner cases that only expose themselves at this scale. I reported one fairly serious problem 10 months ago ( https://github.com/bstansell/conserver/issues/72 ) which remains unresolved to this day.  I've held off on reporting any of the other issues as there didn't seem to be a point if no one is maintaining the codebase, or can provide support.  Unfortunately my area of expertise is not C, so I'm really at a bit of a loss when it comes to debugging or I would have made more of an effort at self-help by now.  I am getting increasing pressure to find creative solutions to these problems, or even consider migrating away from this code base altogether to something that is actively supported (even if that means something that is not open source and/or requires a licensing fee).

**Comment by robohack** [comment]
I would like to say that I think it is always good to report new and/or unique problems, especially in a forum such as this, at least so long as a decent and hopefully clueful report can be made.

That way anyone who does encounter the same problem and/or who has the skills and time/incentive to try to work on it will hopefully have extra information and clues to help them figure out the solution.
